source/implementation.liquid

```diff
@@ -0,0 +1,75 @@
+---
+layout: has_no_sidebars
+title: Implementation Guide
+introduction: >
```

Comment:
@jgerigmeyer I'm not seeing this intro text on the current implementation guide that's [live](https://sass-lang.com/implementation) but I do see it in the code. Do we know if it was left off intentionally or accidentally?

Reply:
Yes, I noticed that too. We can confirm with Natalie in the next demo, but given the text is sensible I think it was likely an oversight? I'll add a note to the card.